convert2rhel/toolopts.py

```diff
@@ -97,40 +129,59 @@ def _get_argparser():
             "  convert2rhel [--no-rhsm] [--disablerepo repoid]"
             " [--enablerepo repoid] [--no-rpm-va] [--debug] [--restart] [-y]\n"
             "  convert2rhel [-k activation_key | -c conf_file_path] [-o organization] [--pool pool_id | -a] [--disablerepo repoid] [--enablerepo"
-            " repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [-y]"
+            " repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [-y]\n"
+            r"  convert2rhel {analyze}"
             "\n\n"
-            "*WARNING* The tool needs to be run under the root user"
+            "*WARNING* The tool needs to be run under the root user\n"
         )
         return argparse.ArgumentParser(
-            prog="convert2rhel",
-            conflict_handler="resolve",
-            usage=usage,
-            add_help=False,
+            conflict_handler="resolve", usage=usage, formatter_class=argparse.RawTextHelpFormatter
```

Comment:
We don't have to use the `argparse.RawTextHelpFormatter` here, as it will break the textwrap for the help message we output. 

```suggestion
            conflict_handler="resolve", usage=usage
```

We can leave this as blank, as the default formatter for argparse will be enough for now.